### PR TITLE
llisence is a Noun/ imporant thing

### PR DESCRIPTION
**Pull request by redmangospros**

### CODE REVIEW

**Comment by redmangospros** [comment]
?

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 ### Always add a README.md
 ![1](./images/3.JPG)
 
-### Use the license feature
+### Use the License feature
```
Comment (vutondesign) [inline README.md]: There isn't really a need for this word to be capitalized as It's part of a title.

Comment (redmangospros) [reply README.md]: Yeah, But You did Pr in my, so please merge this.
Plus, Redeme,md is captial.

**Comment by redmangospros** [comment]
Re-open

File: README.md (hunk 1 above)
Comment (vutondesign) [reply README.md]: That's now how PR's work

Comment (redmangospros) [reply README.md]: Ok. Sorry. 👍

Comment (vutondesign) [reply README.md]: No problem, thanks for taking an interest in my work though,

Comment (redmangospros) [reply README.md]: your welcome, i nned your help

**Pull request closed without merging**